interview-questions.md

```diff
@@ -14,33 +14,33 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Global Information Tracker, it's a system that allows you and other people to work on code together and maintain version control so that you don't have a crazy mess of a tracking system. The true value of git is realized the larger an organization gets.
 
-Researched answer:
+Researched answer: same as above but fancier.  answer.toFancierExplanationOfSimpleConcept()
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The AND &&, OR ||, NOT !, greater than >, less than <, greater than or equal to >=, less than or equal to <=, loosely equal to ==, and strictly equal to === operators are logical operators which give you a result of true or false.
 
-Researched answer:
+Researched answer: A boolean is a datatype that has only two values, true or false. This is important to programming because as code becomes more complicated, the returns will always be one of 6 (or 7 if you count BigInt) primitive datatypes. Being able to set conditionals to the result of a boolean is an integral part of programming.
```

Comment:
nice. Your first answer has examples of all three categories of operators that return the boolean value: equality, relational, and logical